docs/content/cluster-scope/add_operator.md

```diff
@@ -0,0 +1,92 @@
+#  Adding an Operator to a Cluster
+
+There are two methods one can follow to add an Operator to a cluster:
+1. Install via Operator Lifecycle Manager (OLM)
+1. Manual install
+
+Installing via OLM is easier and often the preferred method of installation, but there are a few things to consider when choosing this install method. In order to install your operator via OLM you need to find it in either the [k8s community operators list][1] or [ocp community operators list][2]. If your operator is available in a different catalog source you will need to check if said catalog source is available on the target cluster.
+
+## Install an operator via OLM
+
+### Pre-requisites
+* Basic understanding of k8s and OCP/OLM
+* Basic knowledge of Git and [Kustomize][13]
+* Check if an operator bundle already exists for the desired operator [here][3] (if it does exist you can probably just include this bundle in the desired cluster overlay)
+* Know thename of the operator and its target version, catalogSource, and operatorChannel and what operator channel need to be deployed
+* Know the scope of the operator, is it global or namespace scoped? (NOTE: for non-global operators you will also need to add an `operatorGroup` in the target namespaces, see [here][4] for more info). Global operators are installed to the openshift-operators namespace which already has an `operatorGroup`.
+    * To create a namespace for a non-global operator:
+        * Navigate to: apps/cluster-scope/base/core/namespaces/
+        * Create a directory for the new namespace and include the namespace.yaml and the associated kustomization file
+
+OLM makes it easy to install and manage Operators on OpenShift clusters.
+To install an Operator via OLM, you must do the following:
+
+1. Clone the [apps repo][5]
+1. Navigate to apps/cluster-scope/base/operators.coreos.com/subscriptions/
+    * Create a directory for the new operator and include the subcription.yaml and the associated kustomization file
```

Comment:
I think you want to start with "what does `subscription.yaml` look like?", especially since in the following bullet points you refer to several attributes from the manifest; reading this as a new user might be confusing (include what file where?). Probably also worth spelling `kustomization` as `kustomization.yaml`.